foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/store/ConfigLoader.java

```diff
@@ -181,6 +190,18 @@ public Property read(String groupName, String propertyName) {
         logger.info(String.format("Read property with groupName: %s, propertyName: %s from the store.", groupName, propertyName));
         return propertyDao.read(groupName, propertyName);
     }
-}
-
+    public boolean isFullyLoaded() {
+        Property statusProperty = propertyDao.read("load-status", "fully-loaded");
+        return statusProperty != null && "true".equals(statusProperty.getValue());
+    }
 
+    private void updateLoadStatus(boolean fullyLoaded) {
```

Comment:
Recommend changing this local argument to a different name than `fullyLoaded`, like `status`. Otherwise it confusing to know if `fullyLoaded` is referring to the argument or class attribute.

Reply:
done